dali/pipeline/operator/arg_helper.h

```diff
@@ -291,21 +317,24 @@ class ArgValue {
   /**
    * @brief Read constant argument data
    */
-  int ReadConstant(const OpSpec &spec) {
+  void ReadConstant(const OpSpec &spec, bool error_if_no_value = true) {
```

Comment:
What is the use case for `error_if_no_value=False`?

Reply:
In the constructor we call with with `error_if_no_value=False` and store the value. We don't want to error at that point. It is when we try to Acquire the value and we fall into the constant branch, that we want to raise an error if there's nothing to read.

Reply:
Understood.